libs/qCC_db/include/ccPointCloud.h

```diff
@@ -949,11 +949,20 @@ class QCC_DB_LIB_API ccPointCloud : public CCCoreLib::PointCloudTpl<ccGenericPoi
 	//! Clears the LOD structure
 	void clearLOD();
 
+	//! Return if the cloud has a valuable LOD
+	bool hasUsableLOD() const;
+
+	bool useLODRendering() const;
```

Comment:
Can you comment these new functions and members? (this part of the code is normally rather clean and commented 👼 )